x-pack/plugins/ml/public/application/components/help_menu/index.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, useEffect } from 'react';
+import { i18n } from '@kbn/i18n';
+import { useMlKibana } from '../../contexts/kibana';
+
+interface HelpMenuProps {
+  docLink: string;
+}
+
+export const HelpMenu: FC<HelpMenuProps> = React.memo(({ docLink }) => {
```

Comment:
To be consistent with the other components in the `ml/public/application/components` folder, it is probably better to move this content into `help_menu.tsx` and then create a simple `index.tsx` which exports out `HelpMenu` (copy structure of `ml/public/application/components/create_job_link_card`).